source/adminguide/service_offerings.rst

```diff
@@ -169,65 +163,75 @@ To create a new compute offering:
       system VM is running. Shared allocates from storage accessible via
       NFS.
 
-   -  **Custom**: Custom compute offerings can be used in following
-      cases: deploying a VM, changing the compute offering of a stopped
-      VM and running VMs, which is nothing but scaling up.
+   -  **Provisioning type**: The type of disk that should be allocated. 
+      Local
 
-      If the Custom field is checked, the end-user must fill in the
-      desired values for number of CPU, CPU speed, and RAM Memory when
-      using a custom compute offering. When you check this box, those
-      three input fields are hidden in the dialog box.
+
+   -  **Compute Offering Type**: The amount of freedom that the end user
+      has to customise the compute power that their instance has when using this
+      compute offering.  The options are; Fixed offering - user has no 
+      ability to customise, Custom constrained - user has some latitude
+      to customise the compute within parameters set by the offering, 
+      Custom unconstrained - user can set any values that they wish.
+      'Custom constrained' is recommended over 'Custom unconstrained'
+      
 
    -  **# of CPU cores**: The number of cores which should be allocated
-      to a system VM with this offering. If Custom is checked, this
-      field does not appear.
+      to a system VM with this offering. If 'Custom unconstrained' is checked, this
+      field does not appear. If 'Custom constrained' is checked, the admin will
+      be asked to enter the minimum and maximum number of CPUs that a user
+      can request.
 
-   -  **CPU (in MHz)**: The CPU speed of the cores that the system VM is
+   -  **CPU (in MHz)**: The CPU speed of the cores that the guest instance is
       allocated. For example, “2000” would provide for a 2 GHz clock. If
-      Custom is checked, this field does not appear.
+      Custom is checked, this field does not appear (only enforced if CPU 
```

Comment:
`If 'Custom constrained' or 'Custom unconstrained'` will be better to avoid confusion,

Reply:
done